comp/otelcol/converter/impl/testdata/connectors/dd-connector/config-result.yaml

```diff
@@ -0,0 +1,58 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "datadog-agent"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+exporters:
+  debug:
+    verbosity: detailed
+  datadog:
+    api:
+      key: "0000"
+processors:
+  infraattributes/dd-autoconfigured:
+  batch:
+    timeout: 10s
+connectors:
+  datadog/connector:
+    traces:
+      compute_top_level_by_span_kind: true
+      peer_tags_aggregation: true
+      compute_stats_by_span_kind: true
+extensions:
+  pprof/dd-autoconfigured:
+  health_check/dd-autoconfigured:
+  zpages/dd-autoconfigured:
+    endpoint: "localhost:55679"
+  datadog/dd-autoconfigured:
+
+service:
+  extensions:
```

Comment:
Because here we are not testing the extensions, can you please add these manually like it is done here: https://github.com/DataDog/datadog-agent/blob/main/comp/otelcol/converter/impl/testdata/processors/no-processors/config.yaml#L16-L21. Basically, for anything in testdata I want to isolate the changes to only what is being tested in the specific test case.